docpages/example_programs/interactions_and_components/resolved_objects.md

```diff
@@ -0,0 +1,9 @@
+\page resolved-objects Using Resolved Objects
+
+If your slash command accepts options like user, channel, or role you can get their value, passed by user, from parameters. Though parameter gives you only the snowflake id of the passed value. 
```

Comment:
what does "passed by user" mean here?

Reply:
values user chose when invoking the slash command

Reply:
can it be reworded at all please? perhaps:

you can get their value as specified by the user in the command.

passed by user doesn't really fit

Reply:
yes changed it